test/dummy/app/assets/stylesheets/fenrir_view/extras.scss

```diff
@@ -0,0 +1,16 @@
+//---------------------------------------//
+// This file should stay empty,
+// as it's testing fenrir defaults...
```

Comment:
Comment says the file should be empty but it isn't?

Reply:
Yeah, these bits need to be in there until we remove some older scss code unfortunately.